sdk/nodejs/package.json

```diff
@@ -19,7 +19,7 @@
     "devDependencies": {
         "@types/node": "^10.0.0",
         "@types/semver": "^5.4.0",
-        "typescript": "^2.6.2"
+        "typescript": "^3.0.0"
```

Comment:
this is the primary reason i wanted to do this.  we're using an old enough version of pulumi-terraform that we're referencing versions of TS that don't even understand features created in the last couple of years of TS.